.github/workflows/phpunit_sw_old_versions.yml

```diff
@@ -1,5 +1,5 @@
-# Shopware 5.2 - 5.5 run on PHP versions 5.6, 7.0 and 7.1.
-name: PHPUnit (SW 5.2 - 5.5)
+# Shopware 5.3 - 5.5 run on PHP versions 5.6, 7.0 and 7.1.
+name: PHPUnit (SW 5.3 - 5.5)
```

Comment:
For some reason, it was already removed from the test matrix

Reply:
ok